secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackSecondary.scala

```diff
@@ -385,7 +411,7 @@ abstract class FeedbackSecondary[CT,CV, RCI <: RowComputeInfo[CV]] extends Secon
           val gaveUp = "Gave up replaying updates after too many failed mutation script attempts"
           val newCookie = feedbackCookie.copy(
             current = cookie.copy(
-              computationRetriesLeft = computationHandler.computationRetries, // reset computation retries
+              computationRetriesLeft = computationHandlers.map(_.computationRetries).max, // reset computation retries
```

Comment:
Also make property of FeedbackSecondary and make applicable changes to FeedbackSecondaryInstance